suricata.yaml.in

```diff
@@ -350,6 +352,8 @@ outputs:
       log-dir: files    # directory to store the files
       force-magic: no   # force logging magic on all stored files
       force-md5: no     # force logging of md5 checksums
+      force-sha1: no    # force logging of sha1 checksums
```

Comment:
What if this section was something like:
`force-hash: [sha256] # Available hash functions are [md5, sha1, sha256]`

So if something like SHA3 is added in the future, you can just append the new function, instead of a new variable in the YAML each time.

Further more, instead of having to parse each individual conf node child value (as in https://github.com/serializingme/suricata/blob/dev-feat509-hashing-v0.4/src/log-file.c#L441) this can loop though the array that is found in the "force-hash" node value.